resources/views/components/review/section.blade.php

```diff
@@ -0,0 +1,22 @@
+@props(['reviews','href',])
+
+<div class="flex flex-col gap-3">
+
+<div class="flex justify-between">
+    <h1 class="text-2xl font-bold">Reviews</h1>
+    <x-button variant="secondary" size="sm" href="{{ $href }}">Write a review</x-button>
+</div>
+
+@foreach ($reviews as $review)
+<x-review.index 
+        title="{{ $review->movie->title }}"
+        content="{{ $review->content }}"
```

Comment:
```suggestion
        :title="$review->movie->title"
        :content="$review->content"
```

make sure these are sanitized!